### PR TITLE
Issue/130/grb filters

### PR DESCRIPTION
**Pull request by FusRoman**
**IMPORTANT: Please create an issue first before opening a Pull Request.**
Linked to issue(s): Closes _put link_

## What changes were proposed in this pull request?

How is the issue this PR is referenced against solved with this PR?

### CODE REVIEW

**Comment by JulienPeloton** [comment]
Shall I do the review now finally, or shall I wait?

**Comment by FusRoman** [comment]
Wait until the end of the test-suite and if all is green, you can merge the pull request

**Comment by FusRoman** [comment]
You can do the review now, and I will merge if you say everything is ok.

File: fink_filters/__init__.py
Comment (JulienPeloton) [inline fink_filters/__init__.py:16]: The versioning is done using x.y -- you should put `3.8`

File: fink_filters/filter_on_axis_grb/filter.py
Comment (JulienPeloton) [inline fink_filters/filter_on_axis_grb/filter.py:1]: nit: 2023

File: fink_filters/filter_on_axis_grb/filter.py
Comment (JulienPeloton) [inline fink_filters/filter_on_axis_grb/filter.py:50]: Here there is a potential problem. If SIMBAD is down, you will reject all alerts. 

Alternatively, you can add a guard against downtime of SIMBAD. You could have another filter which is:

```python
f_fail = fink_class.apply(lambda x: x.startswith('Fail'))

f_bronze = f_bogus & (f_class | f_fail)
```

But note that in this case, if SIMBAD is down, only `f_bogus` will apply.

**Review by JulienPeloton — APPROVED** [review]
Thanks @FusRoman ! It is ok for me, but I left a few comments before merging.

**Pull request merged**
